lib/ApplicationFeatures/RocksDBOptionFeature.cpp

```diff
@@ -306,7 +301,7 @@ void RocksDBOptionFeature::validateOptions(
   if (_maxSubcompactions > _numThreadsLow) {
     _maxSubcompactions = _numThreadsLow;
   }
-  if (_blockCacheShardBits > 32) {
+  if (_blockCacheShardBits >= 20) {
```

Comment:
If less than `-1`, issue warning?

Reply:
true ... just add that to this test block and stop.  wild negatives could be sign of something worse since that is so unlikely to happen.

Reply:
Anything < 0 triggers the default: https://github.com/arangodb/arangodb/blob/devel/3rdParty/rocksdb/v5.6.X/cache/lru_cache.cc#L487

Reply:
Agreed, but we should still proof our inputs thoroughly.  (Good work investigating the impact.)

Reply:
you're right, especially because this is cast into an `int` later